stdlib/filepath/src/test/java/org/ballerinalang/stdlib/filepath/nativeimpl/PathTest.java

```diff
@@ -386,8 +385,9 @@ public void testResolveNotLinkPath() {
         assertEquals(returns.length, 1);
         assertTrue(returns[0] instanceof BError);
         BError resolveError = (BError) returns[0];
-        assertEquals(resolveError.getReason(), Constants.NOT_LINK_ERROR);
-        log.info("Ballerina error: " + resolveError.getDetails().stringValue());
+        assertEquals(resolveError.getMessage(),
+                     "Path is not a symbolic link src/test/resources/data-files/test_nolink.txt");
```

Comment:
I doubt this will fail in windows build. shall we remove the file path from the assert

Reply:
Fixed in 6f0cd9f3156126d3a221e576cc91061d5656b530